app/models/move.rb

```diff
@@ -33,6 +33,7 @@ class Move < ApplicationRecord
   belongs_to :to_location, class_name: 'Location', optional: true
   belongs_to :person
   has_many :documents, dependent: :destroy
+  has_many :notifications, as: :topic, dependent: :destroy # NB: polymorphic association
```

Comment:
I'm not sure why this is polymorphic? I don't see the need for it in the current codebase?

Reply:
@starswan it is polymorphic because other classes will have notifications, e.g.  people/profiles